elastic/integrations
# [Synthetics] Fix meta container object mappings

## Proposed commit message



Fix `meta.*` namespace from using dynamic to a `dynamic_template`.

## Checklist


## How to test this PR locally



1. Build integration locally.
2. Install remotely with `elastic-package install`.
3. Check `synthetics-*` mappings have a dynamic template for `meta.*` fields:
```
      "meta": {
        "path_match": "meta.*",
        "mapping": {
          "type": "keyword"
        },
        "match_mapping_type": "string"
      }
```
